CNAME

```diff
@@ -1 +0,0 @@
-personal-portfolio.io
```

Comment:
### Great comments ✨ however, it only lacks a little bit of explanation and it can be improved 🚀 

- [ ] Please ensure to add relevant comments about issues in a piece of code or an enhancement for the project so you can add some reviews since based on that this project will be assessed 👍🏼 .

- [ ] Kindly don't forget about describing to the student the `why` he should make the requested changes so the changes make sense and are fair 👍🏼 💯 

- [ ] Examples are stronger than words in conveying messages🎯, so you can add an example To ensure that the student understands what changes he needs to make 🥇.

- [ ] Add some resources relevant to the request changes so the student can learn something new 📖 .

- [ ] This is **[Optional]** but highly recommended in order for your comments to sound more human-friendly please add some emojis to your comments 🤖 👽

Reply:
Hi @adel-gu 

I made changes to my comments 
please have another look